sparse/pydata_backend/_compressed/compressed.py

```diff
@@ -847,6 +848,16 @@ def __str__(self):
     def ndim(self) -> int:
         return 2
 
+    @classmethod
+    @abstractmethod
+    def get_compressed_axes(cls):
+        pass
```

Comment:
Do we need this method (and the ones in the classes below)?

Reply:
I think yes. `from_coo` that is called in `from_numpy` needs to have `compressed_axes`. If it's `None` then it selects one based on best compression ratio. So there must be this piece of information in `CSC` and `CSR` classes, namely which `compressed_axes` should be used.

Reply:
There's already the `compressed_axes` attribute/property for each instance.

Reply:
But it's an instance attribute - available once object is created. Here we want to make sure that `compressed_axes` is exactly as the class require it (therefore we need a class attribute here for `CSC` and `CSR`, to make sure we have correct `compressed_axes` on creation).

Reply:
Alternatively we could have copied `def from_coo` for `CSR`/`CSC` that just hardcode `compressed_axes` but I think it isn't clean approach. I would stick to class attribute that says what `compressed_axes` is.

Reply:
In that case, let's use a class attribute rather than a class method.

Reply:
Sure - done!